extension/fts/test/test_files/fts_small.test

```diff
@@ -264,11 +264,12 @@ name_stopwords
 ---- error
 Binder exception: Table 0_docIdx_docs does not exist.
 -STATEMENT call show_tables() return *
----- 6
+---- 7
 0|doc|NODE|local(kuzu)|
+0|likes|REL|local(kuzu)|
+1|0_docIdx_appears_in|REL|local(kuzu)|
```

Comment:
Internal table is being printed.

Reply:
Will address internal table related issues in another PR.